db/migrate/20200508155813_populate_response_cached_json.rb

```diff
@@ -35,7 +35,7 @@ def cache_responses(responses)
     total = responses.count
     start = Time.zone.now
     num_procs = ENV["NUM_PROCS"] ? ENV["NUM_PROCS"].to_i : Etc.nprocessors
-    Parallel.each_with_index(responses.find_each, in_processes: num_procs) do |response, index|
+    Parallel.each_with_index(responses.find_each, in_processes: num_procs, isolation: true) do |response, index|
```

Comment:
Note: without `isolation`, global `I18n.locale` is shared across re-used processes.